app/uk/gov/hmrc/awrslookup/views/govuk_wrapper.scala.html

```diff
@@ -80,7 +81,7 @@
 
 @mainContentHeader = {
     @if(!isErrorPage) {
-        @awrsLanguageSelection(clc.languageMap, clc.langToCall(_))
+        @uk.gov.hmrc.awrslookup.views.html.helpers.awrsLanguageSelection(clc.languageMap, clc.langToCall)
```

Comment:
should uk.gov.hmrc.awrslookup.views.html.helpers be imported as it is used multiple times?